app/views/recipes/show.html.erb

```diff
@@ -19,8 +19,8 @@
    <% RecipeFood.includes(:food).where(recipe_id: @recipe.id).each do | recipe_food | %>
     <tr>
    <th><%= recipe_food.food.name %></th>
-   <th><%= recipe_food.quantity %> <%= recipe_food.food.measurement_unit %></th>
-   <th>$<%= recipe_food.food.price %></th>
+   <th><%= recipe_food.food.quantity %> <%= recipe_food.food.measurement_unit %></th>
```

Comment:
- I think it's better to return this value from line 22 to `recipe_food.quantity` as before. This field must present the quantity of the recipe foods table.
<img width="901" alt="Screen Shot 2022-10-06 at 10 10 37 AM" src="https://user-images.githubusercontent.com/47673311/194322181-2f335a42-e70b-451a-a678-f310d01f7d88.png">